DRIVERS/Z055_HDLC/README.md

```diff
@@ -27,6 +26,11 @@ README.PPP            Setting up a PPP network connection (using pppd program)
 *** Applications ***
 ********************
 
+This driver is intended for use in synchronous HDLC mode.
+All packages written to the tty device that this z055_hdlc_drv
+driver creates should contain an entire HDLC frame. Including
```

Comment:
No, not "an entire HDLC frame" as the driver takes care of the flags and checksum. Rather:

The z055_hdlc_drv driver takes the HDLC address and control bytes from the beginning of each data package that is written to the tty device. So all packages written to the tty device should contain those bytes. For example, "echo -ne '\x13\x37Hello world!' > /dev/ttyTH0" will embed "Hello world!" in an HDLC package with address 0x13 and control 0x37.